TestComponentCSharp/Class.cpp

```diff
@@ -1225,6 +1226,26 @@ namespace winrt::TestComponentCSharp::implementation
         return winrt::make<native_properties1>();
     }
 
+    // TODO: when the public WinUI nuget supports IXamlServiceProvider, just use the projection
+    struct __declspec(uuid("68B3A2DF-8173-539F-B524-C8A2348F5AFB")) IServiceProviderInterop : ::IInspectable
```

Comment:
Does ::IInspectable not work throughout.  Do we need WF:: in the other places?

Reply:
I had to #include <inspectable.h>, as a temporary measure to mock winrt::implements<IXamlServiceProvider>, until the public WinUI winmds have that type.  That in turn brought an ambiguous definition of IInspectable into the global namespace, colliding with the C++/WinrT version, so I had to qualify the latter.